webhooks/alias.go

```diff
@@ -69,11 +69,15 @@ func (webhook *MachineDeployment) SetupWebhookWithManager(mgr ctrl.Manager) erro
 }
 
 // MachineSet implements a validating and defaulting webhook for MachineSet.
-type MachineSet struct{}
+type MachineSet struct {
+	Decoder *admission.Decoder
```

Comment:
Let's drop the Decoder field here and in the MachineDeployment struct above

(we don't need it as we can just create a decoder based on the scheme of the manager)

Reply:
✅ updated!